src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMoviesQuery = session.createQuery("FROM Movie", Movie.class);
```

Comment:
there is only one query and it is in `getAll()` method so what it does is obvious from the context
try to use simpler names when purpose is obvious from the context